src/modules/Security.rb

```diff
@@ -373,104 +415,29 @@ def Read
 
       # Read security settings
       read_from_locations
-      Builtins.y2milestone("Settings=%1", @Settings)
+      log.info "Settings=#{@Settings}"
```

Comment:
The logging is inconsistent, here is `info` level, the rest is `debug`. At least the final logging should be `info` to log the final value after reading all parts. And also the message is the same, there should be some context to distinguish the steps, e.g.:

```ruby
log.info "Settings (after reading ...): #{@Settings}"
```